### PR TITLE
Update admin experiment page

### PR DESCRIPTION
**Pull request by tncowart**
Improve Admin pages a bit

### CODE REVIEW

File: cegs_portal/search/admin.py [hunk 1]
```diff
@@ -16,13 +22,48 @@ class DNAFeatureAdmin(admin.ModelAdmin):
...
     list_filter = ["public", "archived"]
     search_fields = ["name", "description"]
 
 
-admin.site.register(Experiment, ExperimentAdmin)
+admin.site.register(Facet)
+
+
+class FacetValueAdmin(admin.ModelAdmin):
+    search_fields = ["value", "facet__name"]
```
Comment (tncowart) [inline cegs_portal/search/admin.py:63]:
```suggestion
class FacetValueAdmin(admin.ModelAdmin):
    search_fields = ["value", "facet__name"]
    list_display = ["facet_list_display"]

    @admin.display
    def facet_list_display(self, obj):
        return f"{obj.value} ({obj.facet.name})"
```

File: cegs_portal/search/admin.py [hunk 2]
```diff
@@ -1,8 +1,14 @@
+from django import forms
 from django.contrib import admin
+from django.db import models
```
Comment (tncowart) [inline cegs_portal/search/admin.py:3]: Remove unused imports

File: cegs_portal/search/admin.py [hunk 3]
```diff
@@ -16,7 +23,73 @@ class DNAFeatureAdmin(admin.ModelAdmin):
 admin.site.register(DNAFeature, DNAFeatureAdmin)
 
 
+class OtherFileForm(forms.ModelForm):
```
Comment (tncowart) [inline cegs_portal/search/admin.py:26]: This seems unnecessary right now -- It doesn't do anything as far as I can tell. I think you should remove it.

File: cegs_portal/search/admin.py [hunk 4]
```diff
@@ -16,7 +23,73 @@ class DNAFeatureAdmin(admin.ModelAdmin):
...
+    #     widgets = {"description": forms.Textarea(attrs={"rows": 6, "columns": 90})}
+
+
+class ExperimentFacetValueForm(forms.ModelForm):
+    pass
+    # class Meta:
+    #     widgets = {"description": forms.Textarea(attrs={"rows": 6, "columns": 90})}
+
+
+class ExperimentDataFileForm(forms.ModelForm):
```
Comment (tncowart) [inline cegs_portal/search/admin.py:38]: This seems unnecessary right now -- It doesn't do anything as far as I can tell. I think you should remove it.

File: cegs_portal/search/admin.py (hunk 2 above)
Comment (tncowart) [reply cegs_portal/search/admin.py:3]: These unused imports are still there.

Comment (UAMS16) [reply cegs_portal/search/admin.py:3]: Imports plural? Looks like I can remove "from django.contrib import admin". May need the rest.

Comment (tncowart) [reply cegs_portal/search/admin.py:3]: Neither of these are used:
```python
from django.db import models
from django.forms import Textarea
```

Comment (UAMS16) [reply cegs_portal/search/admin.py:3]: Yes, you are right. Thought I needed it for using textarea in the code but forms seem to handle that.

**Pull request merged**